storage/container_storage_interface/persistent-storage-csi-migration.adoc

```diff
@@ -0,0 +1,59 @@
+[id="persistent-storage-csi-migration"]
+= CSI Automatic Migration
+
+toc::[]
+
+Starting with {product-title} 4.8, automatic migration for the following in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers is available:
+
+* Amazon Web Services (AWS) Elastic Block Storage (EBS)
+
+* OpenStack Cinder
+
+:FeatureName: CSI Automatic Migration
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+With automatic migration enabled, volumes provisioned using in-tree storage plug-ins supported by this feature are migrated to their counterpart CSI drivers. By default, automatic migration is disabled.
+
+.Procedure
+
+You can enable and disable CSI automatic migration using feature gates (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling features using feature gates]).
```

Comment:
A procedure is usually a series of one or more tasks. Can you make this task-based? Something like:

To enable migration for all CSI drivers supported by CSI automatic migration:

- Add the following `featureSet` parameter using , as described in [Enabling features using feature gates](https://deploy-preview-31774--osdocs.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-enabling-features.html):

```
[source,yaml]
----
apiVersion: config.openshift.io/v1
kind: FeatureGate
metadata:
  name: cluster
spec:
  featureSet: TechPreviewNoUpgrade <1>
...
----
```

<1> Specifying `TechPreviewNoUpgrade` enables migration for all CSI drivers supported by CSI automatic migration.